Content.Shared/Singularity/Components/EventHorizonComponent.cs

```diff
@@ -62,7 +63,7 @@ public sealed class EventHorizonComponent : Component
     public TimeSpan TargetConsumePeriod { get; set; } = TimeSpan.FromSeconds(0.5);
 
     /// <summary>
-    /// The last time at which this consumed everything it overlapped with.
+    /// The next time at which this consumes everything it overlaps with.
```

Comment:
Pretty sure you corrected the wrong one. The next wave time is the other one.

Reply:
Also, since you appear to have removed all of the references to this you can probably axe it.

Reply:
I think I fucked some shit in my merge but all deleted now.